checker/tests/nullness/generics/MyMap.java

```diff
@@ -1,6 +1,8 @@
 import java.util.Map;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
+// @skip-test issue with @KeyFor
```

Comment:
checker/tests/nullness/generics/MyMap.java:16: error: [enhancedfor.type.incompatible] incompatible types in enhanced for loop.
```
  found   : @UnknownKeyFor Entry<@KeyFor("map") ?[ extends K[ extends @KeyFor("map") Object super @KeyFor("map") Void] super @KeyFor("map") Void], ?[ extends V[ extends @UnknownKeyFor Object super @KeyForBottom Void] super @KeyForBottom Void]>
  required: @UnknownKeyFor Entry<?[ extends K[ extends @UnknownKeyFor Object super @KeyForBottom Void] super @KeyForBottom Void], ?[ extends V[ extends @UnknownKeyFor Object super @KeyForBottom Void] super @KeyForBottom Void]>
```